extensions/ocaml/extension.toml

```diff
@@ -1,7 +1,7 @@
 id = "ocaml"
 name = "OCaml"
 description = "OCaml support."
-version = "0.0.1"
+version = "0.0.2"
```

Comment:
Can we revert the version bump?

We try to keep these separate from the changes in extensions.

I would do it myself, but unfortunately GitHub doesn't let maintainers push to a PR opened from a fork 😕